doc/source/examples/multiphysics/sloshing-in-rectangular-tank/sloshing-in-rectangular-tank.rst

```diff
@@ -0,0 +1,189 @@
+
+    # --------------------------------------------------
+    # Simulation Control
+    #---------------------------------------------------
+    subsection simulation control
+      set method           = bdf2
+      set time end         = 50
+      set time step        = 0.01
+      set adapt            = true
+      set max cfl          = 0.25
+      set output name      = sloshing-in-rectangular-tank_Re20
+      set output path      = ./output_Re20/
+      set output frequency = 1
+      set adaptative time step scaling = 1.025
+    end
+
+The ``multiphysics`` subsection is used to enable the VOF solver.
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Multiphysics
+    #---------------------------------------------------
+    subsection multiphysics
+      set VOF  = true
+    end 
+    
+
+In the ``initial condition``, we define the initial height of the wave ensuring that the interface (the 0.5 isovalue) lies at the right height.
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Initial condition
+    #---------------------------------------------------
+    subsection initial conditions
+      set type = nodal
+      subsection uvwp
+        set Function expression = 0; 0; 0
+      end
+    
+      subsection VOF
+        set Function expression =  if (y<=(0.01*sin(3.1416*(x+0.5))), min(0.5-(y-0.01*sin(3.1416*(x+0.5)))/0.0025,1), max(0.5-(y-0.01*sin(3.1416*(x+0.5)))/0.0025,0))
+        
+        subsection projection step
+          set enable           = false
+          set diffusion factor = 1
+        end
+        
+      end
+    end
+
+In the ``mesh`` subsection, we define an hyper rectangle with the appropriate dimensions. The mesh is initially refined 6 times to ensure an adequate definition of the interface.
+
+.. code-block:: text
+
+  #---------------------------------------------------
+  # Mesh
+  #---------------------------------------------------
+
+  subsection mesh
+    set type               = dealii
+    set grid type          = subdivided_hyper_rectangle
+    set grid arguments     = 5, 2 : -1, -1 : 1, 0.1 : true
+    set initial refinement = 6
+  end
+
+
+The ``physical properties`` are mainly used to establish the Reynolds number of the sloshing liquid. For the air, however, the work of Carrica does not give any physical properties. We thus fix the air to be significantly less dense, but we keep the viscosity of the air at a certain reasonable viscosity to ensure numerical stability.
+
+.. code-block:: text
+
+  #---------------------------------------------------
+  # Physical Properties
+  #---------------------------------------------------
+  
+  subsection physical properties
+    set number of fluids = 2
+    subsection fluid 0
+      set density             = 0.001
+      set kinematic viscosity = 0.001
+    end
+    subsection fluid 1
+      set density             = 1
+      set kinematic viscosity = 0.5
+    end
+  end
+
```

Comment:
Would it be interesting to add the `Source term` subsection, since it is the main driving force of the problem?

Reply:
Why not, I can add it.